tests/integration/conftest.py

```diff
@@ -133,6 +133,12 @@ def netbox_docker_repo_dirpaths(pytestconfig, git_toplevel):
             subp.check_call(
                 ["git", "fetch"], cwd=repo_fpath, stdout=subp.PIPE, stderr=subp.PIPE
             )
+            subp.check_call(
+                ["git", "reset", "--hard"],
+                cwd=repo_fpath,
+                stdout=subp.PIPE,
+                stderr=subp.PIPE,
+            )
```

Comment:
This was needed to fix some changes that were made when I ran black over everything.. it's probably just a good idea in general though.